runtime/tr.source/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -5814,7 +5814,7 @@ J9::Z::TreeEvaluator::VMgenCoreInstanceofEvaluator2(TR::Node * node, TR::CodeGen
                if (cg->needClassAndMethodPointerRelocations())
                   temp = generateRegLitRefInstruction(cg, TR::InstOpCode::getLoadOpCode(), node, arbitraryClassReg1, (uintptrj_t) profiledClassesList[numPICs].profiledClass, TR_ClassPointer, NULL, NULL, NULL);
                else
-                  temp = generateRILInstruction(cg, TR::InstOpCode::LARL, node, arbitraryClassReg1, (uintptrj_t)profiledClassesList[numPICs].profiledClass);
+                  temp = generateRILInstruction(cg, TR::InstOpCode::LARL, node, arbitraryClassReg1, profiledClassesList[numPICs].profiledClass);
```

Comment:
Is this supposed to use the `void*` overload? Seems to be loading an address? Similarly on the next change using `profiledClassesList`.

Reply:
Similarly as above `profiledClass` is a  `TR_OpaqueClassBlock*` so this will call the `void*` overload as intended.